docs/_static/nuts-registry.yaml

```diff
@@ -314,6 +350,36 @@ paths:
                       $ref: '#/components/schemas/Event'
 components:
   schemas:
+    CAListWithChain:
```

Comment:
- Not sure this structure is still logical when the Root CA certificate is renewed and thus there are multiple roots?
- Not sure what the use case is to return them splitted (Root CA + Root party's intermediates v.s. Vendor CA certs), instead of just a single list?

Reply:
The main goal is to add CAs to a PEM file. For this we'll need all active roots. Multiple can be listen for sure.
Single list is also supported as application/x-pem-file output. But a vendor might want to do some additional checks before applying them to a reverse proxy. (like activate/deactivate on command)